02LessonTwo/in-class-project/index.html

```diff
@@ -5,6 +5,21 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+<div style="color: #1E8449">
+  <img src="https://www.worldwidegolfshops.com/Assets/images/storeBrands/rd-santaana-1.jpg"/>
+  <h1>Trevino's Golf Store</h1>
+      <div>
+        <h2>Our Online Store</h2>
+        <p> We have all the things</p>
+        <p>We have:</p>
+        <ul>
+            <li> Item 1</li>
```

Comment:
next time be more creative and use real words, that will make your site look better